extensions/wikia/AdEngine/js/slot/service/srcProvider.js

```diff
@@ -18,6 +20,10 @@ define('ext.wikia.adEngine.slot.service.srcProvider', [
 			originalSrc = 'premium';
 		}
 
+		if (babDetection.isBlocking()) {
+			originalSrc = getRecoverySrc();
```

Comment:
Should it be returned here? Correct me if I'm wrong, but now we're going to have `test-test-rec`

Reply:
getRecoverySrc is used in googleTag.js:154
Why do we need to handle src here again?

Reply:
You right, i removed `addTestPrefixForTestWiki()` from `getRecoverySrc()`.

Function `getRecoverySrc()` is not used anywhere, event before cleanup - it's some old code.